interface/modules/custom_modules/oe-module-weno/templates/download_log_viewer.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+require_once(dirname(__DIR__, 4) . "/globals.php");
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Core\Header;
+use OpenEMR\Modules\WenoModule\Services\WenoLogService;
+
+if (!AclMain::aclCheckCore('admin', 'super')) {
+    // renders in MM iFrame
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Must be an Admin")]);
+    exit;
+}
+
+$logService = new WenoLogService();
+$pres_log = $logService->getLastPrescriptionLogStatus();
+$pharm_log = $logService->getLastPharmacyDownloadStatus();
+
+$startDate = $_GET['startDate'] ?? date('m/d/Y'); // just default to today
+$endDate = $_GET['endDate'] ?? date('m/d/Y');
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><?php echo xlt('Weno Downloads'); ?></title>
+    <?php Header::setupHeader(['datetime-picker']); ?>
+    <style>
+      .hide {
+        display: none;
+      }
+    </style>
+    <script>
+        $(function () {
+            $('.datepicker').datetimepicker({
+                <?php $datetimepicker_timepicker = false; ?>
+                <?php $datetimepicker_showseconds = false; ?>
+                <?php $datetimepicker_formatInput = true; ?>
```

Comment:
since not supporting internationalization of the dates in this script, would make above false (or else things will break if using a non YYYY-mm-dd date)

Reply:
nice catch @bradymiller saved me some grief!